src/kafka-to-nexus.cxx

```diff
@@ -33,12 +33,19 @@ int main(int argc, char **argv) {
            "\n"
            "  --config-file               <filename.json>\n"
            "\n"
-           "  --broker-command            <//host[:port][/topic]>\n"
-           "      Kafka brokers to connect with for configuration updates.\n"
+           "  --command-uri               <//host[:port][/topic]>\n"
```

Comment:
These changes look good, but `std::cout` should be used instead of `printf` in C++